test/automation/src/positron/positronNotebooks.ts

```diff
@@ -23,6 +24,7 @@ const PYTHON_OUTPUT = '.output-plaintext';
 const R_OUTPUT = '.output_container .output';
 const REVERT_AND_CLOSE = 'workbench.action.revertAndCloseActiveEditor';
 const MARKDOWN_TEXT = '#preview';
+const activeRowSelector = `.notebook-editor .monaco-list-row.focused`;
```

Comment:
Any reason why you didn't stick with all caps for this one?

Reply:
Yes, I blindly copied and pasted it, LOL.